lib/yu.rb

```diff
@@ -250,7 +250,10 @@ def template_dir
       relative_path.realpath
     end
 
-    def run_command(command, showing_output: true, exit_on_failure: true)
+    def run_command(command, options={})
```

Comment:
Doesn't ruby 2.0 have named params with default values? I don't think this needs changing